components/dashboard/src/teams/OrgSettingsPage.tsx

```diff
@@ -43,7 +43,7 @@ export function OrgSettingsPage({ children }: OrgSettingsPageProps) {
     const subtitle = "Manage your organization's settings.";
 
     // Render as much of the page as we can in a loading state to avoid content shift
-    if (org.isLoading || orgBillingMode.isLoading) {
+    if (org.isLoading) {
```

Comment:
With `orgBillingMode.isLoading`, collaborator `/billing` router (TeamUsageBasedBilling.tsx) will keep re-rendering forever.

Don't know what side-affect this change will cause. But if we consider collaborator should not visit /billing, then we can revert code here.